src/components/AppRoutes.jsx

```diff
@@ -9,18 +9,22 @@ import UsersUpdate from './Users/UsersUpdate';
 import Product from './Product/ProductList';
 import Navigation from './Navigation/Navigation';
 import WIP from './shared/WIP';
+import TransactionList from './Transaction/TransactionList';
+import AddTransaction from './Transaction/AddTransaction';
+import history from './Transaction/history';
 
 export default function AppRoutes() {
   return (
-    <Navigation>
+    <Navigation history={history}>
```

Comment:
Why do we need this? I don't think it's necessary

Reply:
I tried to redirect to a new page without included in the AppRoutes.jsx. I find this is an easy way to redirect to a new page,  so I added this.

Reply:
You can use the [useHistory hook](https://reactrouter.com/web/api/Hooks/usehistory)

And then simply `history.push()` 

You don't actually need to pass in history or install this module

Reply:
Thank you, I will change this.